concepticondata/conceptlists/Anonby-2018-1500.tsv

```diff
@@ -0,0 +1,1501 @@
+Anonby-2018-1500-1192	1192	2194	FRONT (PART)	front (of something)	جلو	jelaʋ / jolo	Environment	Space and objects	
+Anonby-2018-1500-1193	1193	2323	BACK (PART)	back (of something)	پشت، عقب	pošt, aġab	Environment	Space and objects	
+Anonby-2018-1500-1194	1194	682	SIDE	side (of something)	کنار	kenār	Environment	Space and objects	
+Anonby-2018-1500-1195	1195	1093	MIDDLE	middle	وسط، میان	ʋasat, miyān	Environment	Space and objects	
+Anonby-2018-1500-1196	1196	863	EDGE	edge (n.)	لبه	labe	Environment	Space and objects	
+Anonby-2018-1500-1197	1197			point (n.)	نوک	nok	Environment	Space and objects	
+Anonby-2018-1500-1198	1198			bump (n.), protuberance	بر آمدگی	barāmadegi	Environment	Space and objects	
+Anonby-2018-1500-1199	1199	519	SPOT	spot (n.)	نقطه، نقطه و جای مشخص	noġte, noġte ʋa jā-ye mošaxas	Environment	Space and objects	
+Anonby-2018-1500-1200	1200	2874	MOVE (ONESELF)	move (intr.)	حرکت کردن، تکان خوردن	harkat / harekat kardan, takān xordan	Events and actions	Movement (mostly intransitive)	
+Anonby-2018-1500-1201	1201	3695	MOVEMENT	movement	حرکت	harkat / harekat	Events and actions	Movement (mostly intransitive)	
+Anonby-2018-1500-1202	1202	1446	COME	come	آمدن	āmadan	Events and actions	Movement (mostly intransitive)	
+Anonby-2018-1500-1203	1203	695	GO	go	رفتن	raftan	Events and actions	Movement (mostly intransitive)	
+Anonby-2018-1500-1204	1204	1778	APPROACH	approach (v.)	نزدیک شدن	nazdik šodan	Events and actions	Movement (mostly intransitive)	
+Anonby-2018-1500-1205	1205	1387	ARRIVE	arrive	رسیدن	residan	Events and actions	Movement (mostly intransitive)	
+Anonby-2018-1500-1206	1206	987	REMAIN	remain, stay	ماندن	māndan	Events and actions	Movement (mostly intransitive)	
+Anonby-2018-1500-1207	1207	1752	LEAVE	leave (place)	ترک کردن	tark kardan	Events and actions	Movement (mostly intransitive)	
+Anonby-2018-1500-1208	1208	581	COME BACK	return, go back	بر گشتن، بازگشتن	bargaštan, bāzgaštan	Events and actions	Movement (mostly intransitive)	
+Anonby-2018-1500-1209	1209	749	ENTER	enter, go in	وارد شدن	ʋāred šodan	Events and actions	Movement (mostly intransitive)	
+Anonby-2018-1500-1210	1210	1107	GO OUT	come (or go) out, exit (v.)	خارج شدن، بیرون رفتن	xārej šodan, birun raftan	Events and actions	Movement (mostly intransitive)	
+Anonby-2018-1500-1211	1211	705	GO UP (ASCEND)	ascend, go up	بالا رفتن	bālā raftan	Events and actions	Movement (mostly intransitive)	
+Anonby-2018-1500-1212	1212	2014	GO DOWN (DESCEND)	descend, go down	پایین آمدن	pāyin āmadan	Events and actions	Movement (mostly intransitive)	
+Anonby-2018-1500-1213	1213	1280	FALL	fall (intr.)	افتادن	oftādan	Events and actions	Movement (mostly intransitive)	
+Anonby-2018-1500-1214	1214	2871	SWING (MOVEMENT)	swing (v.), go back and forth	تاب خوردن	tāb xordan	Events and actions	Movement (mostly intransitive)	
+Anonby-2018-1500-1215	1215	1324	SLIP	slide	سر خوردن، لغزیدن	sor xordan, laġzidan	Events and actions	Movement (mostly intransitive)	
```

Comment:
I would unmap because SLIP implies stumbling and (nearly) falling while here, at least judging by the English gloss, there is no such implication.